Source/UINavigation/Private/UINavWidget.cpp

```diff
@@ -3105,7 +3109,7 @@ void UUINavWidget::HoverEvent(int Index)
 		return;
 	}
 
-	if (UINavPC->GetCurrentInputType() != EInputType::Mouse || Index == ButtonIndex)
+	if (Index == ButtonIndex || !UHeadMountedDisplayFunctionLibrary::IsHeadMountedDisplayEnabled() && (UINavPC->GetCurrentInputType() != EInputType::Mouse && (!bUseLeftThumbstickAsMouse || !UINavPC->IsMovingLeftStick())))
```

Comment:
You need to separate the IsHeadMountedDisplay boolean into its own local variable, so you don't have to call it if you're not using a VR platform

Reply:
Fixed.